README.md

```diff
@@ -12,12 +12,59 @@ The only IRC Client you'll ever need, guaranteed!
 ## Missing Features
 
 Check the [issues](http://github.com/Chewsterchew/DIRCord/issues) to see what I'm planning next, or to suggest new things!
+## Requirements
 
-## Setup guide.
+* [Ruby (>= 2.4)](https://www.ruby-lang.org/en/)
+* [RVM](https://rvm.io/)
```

Comment:
Not a requirement

Reply:
You told me that RVM was needed the other day.